app/src/main/java/com/ubergeek42/WeechatAndroid/utils/History.kt

```diff
@@ -0,0 +1,54 @@
+package com.ubergeek42.WeechatAndroid.utils
+
+import android.os.Bundle
+import android.text.Editable
+import android.widget.EditText
+import androidx.core.os.bundleOf
+import com.ubergeek42.WeechatAndroid.service.P
+
+class History {
+    private var index = -1
```

Comment:
Hard to understand what this is, also what other fields are

Reply:
Meh I don't agree, it's an index. Renamed anyway.